packages/lib/src/services/player-sdk.service.ts

```diff
@@ -173,6 +173,13 @@ export class PlayerSdk {
     this.set('captionTrack', lang);
   }
 
+  /**
+   * Gets the available audience reactions
+   */
+  async getAudienceReactions(): Promise<string[]> {
```

Comment:
Shouldn't we call it `getAvailableAudienceReactions` or `getAudienceReactionsAvailable`? 
Now it suggests we can get all the reactions from the current event.

Reply:
I do not know if that is necessary, it is always in the context of the current event, so you will get the ones set by the creator.

Reply:
I guess we will never want to get any other context of reactions from the sdk level, so let it be.